pages/thank-you.jsx

```diff
@@ -3,11 +3,10 @@
 
 import React from 'react';
 import MozillaFooter from '../components/footer-mozilla.jsx';
-import Header from '../components/header.jsx';
 import Signup from '../components/signup.jsx';
 import Social from '../components/social.jsx';
 import ThankYouHeader from '../components/thank-you-header.jsx';
-import { FormattedHTMLMessage, IntlMixin } from 'react-intl';
```

Comment:
`FormattedHTMLMessage` was only used in the baseline markup, so I removed it here.